src/main/java/io/github/externschool/planner/entity/Authority.java

```diff
@@ -12,6 +12,9 @@
     @Column
     private String name;
 
+    @ManyToMany(mappedBy = "authorities")
+    private List<User> users = new ArrayList<>();
```

Comment:
your field is not final, no need to initialize it here

Reply:
I think this code is redundant.

We don't want get list of users by authority.

We want to get list of authorities by users

link from user is enough